core/Http.php

```diff
@@ -260,6 +260,12 @@ public static function sendHttpRequestBy(
             }
         }
 
+        // When sending an insecure request, but https is forced, and we would care about valid certificates, log a warning
+        // Note: accepting invalid ssl certificates should only be used when requesting data from a configured website
+        if ($parsedUrl['scheme'] === 'http' && SettingsPiwik::isHttpsForced() && $acceptInvalidSslCertificate === true) {
```

Comment:
Should this be `acceptInvalidSslCertificate === false` to match "in case we do not care about valid certificates we do not log"?

Reply:
Good catch. That's right
```suggestion
        if ($parsedUrl['scheme'] === 'http' && SettingsPiwik::isHttpsForced() && $acceptInvalidSslCertificate === false) {
```